### PR TITLE
Implement chat feature in Station Page

### PR DESCRIPTION
**Pull request by pvtri96**
Refer to #177

### CODE REVIEW

File: backend/src/socket/events/addStationChat.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import * as stationController from '../../controllers/station';
+import * as userController from '../../controllers/user';
+import * as EVENTS from '../../const/actions';
```
Comment (hynguyen2610) [inline backend/src/socket/events/addStationChat.js]: Why it is "EVENT" but not event?

Comment (pvtri96) [reply backend/src/socket/events/addStationChat.js]: Hi @hynguyen2610 

I am following the code convention from other modules. Every action in backend side called EVENTS and  `uppercase`. You can take a look at other modules in `backend/src/socket/events`.

Comment (pvtri96) [reply backend/src/socket/events/addStationChat.js]: I think that they are constants, so it is better to import as `EVENTS`

Comment (hynguyen2610) [reply backend/src/socket/events/addStationChat.js]: Alright ;)

**Pull request merged**